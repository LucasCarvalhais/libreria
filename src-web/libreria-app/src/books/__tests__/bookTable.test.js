import React from 'react';
import renderer from 'react-test-renderer';
import { BookTable } from '../BookTable';

const props = {
    books: [
        { bookId: 1, title: 'teste1', description: 'teste1', author: 'Lucas', edition: 1 },
        { bookId: 2, title: 'teste2', description: 'teste2', author: 'Lucas', edition: 1 },
        { bookId: 3, title: 'teste3', description: 'teste3', author: 'Lucas', edition: 1 }, 
    ],
    isLoading: false,
};

test('bookTable has a valid snapshot', () => {
    const component = renderer.create(<BookTable {...props} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
