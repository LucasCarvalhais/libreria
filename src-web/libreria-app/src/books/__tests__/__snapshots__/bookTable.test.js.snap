// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bookTable has a valid snapshot 1`] = `
<div>
  <div>
    <table
      className="bookTable"
    >
      <tbody>
        <tr>
          <th>
            Código
          </th>
          <th>
            Título
          </th>
          <th>
            Descripción
          </th>
          <th>
            Persona autora
          </th>
          <th>
            Edición
          </th>
        </tr>
        <tr>
          <td>
            1
          </td>
          <td>
            teste1
          </td>
          <td>
            teste1
          </td>
          <td>
            Lucas
          </td>
          <td>
            1
          </td>
        </tr>
        <tr>
          <td>
            2
          </td>
          <td>
            teste2
          </td>
          <td>
            teste2
          </td>
          <td>
            Lucas
          </td>
          <td>
            1
          </td>
        </tr>
        <tr>
          <td>
            3
          </td>
          <td>
            teste3
          </td>
          <td>
            teste3
          </td>
          <td>
            Lucas
          </td>
          <td>
            1
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
